ui/tests/integration/components/open-api-explorer/swagger-ui-test.js

```diff
@@ -57,4 +55,15 @@ module('Integration | Component | open-api-explorer | swagger-ui', function (hoo
     // if the search fn breaks, this test will fail
     assert.dom(SELECTORS.searchInput).hasValue('token', 'search input has value');
   });
+
+  test('it should render camelized operation ids', async function (assert) {
```

Comment:
Could we do a test to ensure that no IDs show in the production environment? Just in case a future change accidentally changes this. It's also documented that the intended behavior is to _not_ show these outside of development.

Reply:
Sure thing just pushed one up!